app/build.gradle

```diff
@@ -50,6 +50,8 @@ dependencies {
     implementation 'com.facebook.android:facebook-core:8.1.0'
     implementation 'com.facebook.android:facebook-login:8.1.0'
 
+    implementation 'com.google.code.gson:gson:2.8.6'
+
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     implementation 'androidx.core:core-ktx:1.3.2'
     implementation 'androidx.appcompat:appcompat:1.2.0'
```

Comment:
use variable for version
ex
def fb_version = '8.1.0'
implementation "com.facebook.android:facebook-core:$fb_version"